core/rawdb/freezer.go

```diff
@@ -270,46 +270,49 @@ func (f *Freezer) ModifyAncients(fn func(ethdb.AncientWriteOp) error) (writeSize
 }
 
 // TruncateHead discards any recent data above the provided threshold number, only keep the first items ancient data.
-func (f *Freezer) TruncateHead(items uint64) error {
+func (f *Freezer) TruncateHead(items uint64) (uint64, error) {
```

Comment:
Can we also add descriptions for the returning old head number here? Just like for the TruncateTail.

Reply:
oke let me update